app/views/categories/index.html.erb

```diff
@@ -1,17 +1,34 @@
-<div class='category-list' itemscope itemtype='http://schema.org/ItemList'>
-  <meta itemprop='itemListOrder' content='http://schema.org/ItemListOrderDescending'>
-  <% @category_list.categories.each_with_index do |c, index| %>
-    <div class='category' itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
-      <meta itemprop='position' content='<%= index %>'>
-      <meta itemprop='url' content='<%= "#{c.url}" %>'>
-      <h2>
-        <a href='<%= "#{c.url}" %>' itemprop='item'>
-          <span itemprop='name'><%= c.name %></span>
-        </a>
-      </h2>
-      <span itemprop='description'><%= c.description&.html_safe %></span>
-    </div>
-  <% end %>
+<div itemscope itemtype='http://schema.org/ItemList'>
+  <table class='category-list'>
+    <thead>
+      <tr>
+        <th class='category'><%= t 'js.categories.category' %></th>
+        <th class='topics'><%= t 'js.topic.list' %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <meta itemprop='itemListOrder' content='http://schema.org/ItemListOrderDescending'>
+      <% @category_list.categories.each_with_index do |c, index| %>
```

Comment:
You don't need to keep track of the index just to set an odd/even class. You can do that using CSS only 😉 

https://www.w3.org/Style/Examples/007/evenodd.en.html

Reply:
Haha, actually I didn't add an index in the loop. It's already used in this line: ` <meta itemprop='position' content='<%= index %>'>` , we need it for SEO @ZogStriP

Reply:
My bad, I missed it. You should still use CSS to do the odd/even class ;)

Reply:
Haha, I have made the change @ZogStriP

Reply:
Sorry, I just looked at your screenshot and I wonder: why aren't we using the category color for the border?

Reply:
Actually, I didn't know that the category had colour stored. I have fixed it now, thanks @ZogStriP  :bowing_man: